sympy/algebras/quaternion.py

```diff
@@ -84,6 +112,158 @@ def d(self):
     def real_field(self):
         return self._real_field
 
+    @classmethod
+    def from_euler(cls, angles, seq):
+        """Returns quaternion equivalent to Euler angles represented same in
```

Comment:
```suggestion
        """Returns quaternion equivalent to Euler angles represented in
```

Reply:
Oops, that's indeed a typo! I meant to say `Returns quaternion equivalent to rotation represented by the Euler angles`.